src/Swarm/Game/Entity.hs

```diff
@@ -162,6 +169,22 @@ newtype GrowthTime = GrowthTime (Integer, Integer)
 defaultGrowthTime :: GrowthTime
 defaultGrowthTime = GrowthTime (100, 200)
 
+-- | Properties of combustion
+data Combustibility = Combustibility {
+    ignition :: Double
+    -- ^ Likelihood of ignition by a neighbor, per tick.
+    -- This gets multiplied by the number of (4-adjacent) neighbors
```

Comment:
This doesn't sound right, and it's not what the code actually does.  If you have 4 neighbors burning your probability of igniting is not 4 times the probability with only one neighbor burning.  It's actually something like 1 - (1 - p)^4, but we don't need to put that in a comment.

Reply:
You caught me with a "draft" PR! :satisfied: 😇 

But the attention is appreciated anyway.

Reply:
Oh, I knew it was a draft, just thought I'd take a look and leave a few comments.  If I suggest doing something you were already going to do, or make a comment about part of the code that's going to change anyway, feel free to ignore me. :smile: